spark/v3.4/spark/src/test/java/org/apache/iceberg/spark/actions/TestCreateActions.java

```diff
@@ -521,17 +535,13 @@ public void testProperties() throws Exception {
     expectedProps.put("dogs", "sundance");
 
     for (Map.Entry<String, String> entry : expectedProps.entrySet()) {
-      Assert.assertTrue(
-          "Created table missing property " + entry.getKey(),
-          table.properties().containsKey(entry.getKey()));
-      Assert.assertEquals(
-          "Property value is not the expected value",
-          entry.getValue(),
-          table.properties().get(entry.getKey()));
+      assertThat(table.properties())
```

Comment:
nit: The outer `for` loop looks redundant. We could use `containsAllEntriesOf` instead.

Reply:
I missed this part, thanks. Also update the Spark 3.5 test.